CHANGELOG.md

```diff
@@ -6,9 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- Feature: Add Operating System flexibility in script (i.e download/install packages depending on OS flavor)
-- Feat: pair different subnet IDs with their corresponding CIDRs for consistency in examples usage
+- feat: add cloud-init script for creating os users
+- feat: Separate the installation process of ssm agent and cloudwatch agent
+- feat: pair different subnet IDs with their corresponding CIDRs for consistency in examples usage
 - fix: Ensure packages are installed without public IP
+- feat: Include ssm installation option for windows instances
+- fix: CKV_AWS_290: "Ensure IAM policies does not allow write access without constraints"
+- fix: CKV_AWS_355: "Ensure no IAM policies documents allow "*" as a statement's resource for restrictable actions"
+
+##  [2.0.0] - 2023-07-19
+### Changes
+- feat: Added ssm support and removed key pair creation for different linux distros
+- feat: Add Operating System flexibility in script (i.e download/install packages depending on OS flavor)
```

Comment:
The change on the dynamic CPU configuration is missing, this is also a breaking change so we should mention it